File: app/src/main/java/com/numero/sojodia/model/BusTime.java
package com.numero.sojodia.model;

public class BusTime {
    public int hour, min, week;
    public boolean isNonstop;

    public BusTime(int hour, int min, int week, boolean isNonstop){
        this.hour = hour;
        this.min = min;
        this.week = week;
        this.isNonstop = isNonstop;
    }
}

File: app/src/main/kotlin/com/numero/sojodia/model/BusTime.kt
package com.numero.sojodia.model

import com.squareup.moshi.Json
import se.ansman.kotshi.JsonSerializable

@JsonSerializable
data class BusTime(
        val hour: Int,
        val minute: Int,
        @Json(name = "week")
        val weekId: Int,
        val isNonstop: Boolean,
        val isOnlyOnSchooldays: Boolean) {

    val time: Time = Time(hour, minute)

    val week: Week? = Week.getWeek(weekId)
}
